### DISCUSSION TITLE
mogrify with colormap (hald-clut)

### DISCUSSION THREAD

**Discussion by teridon** [ImageMagick/ImageMagick#6177] [Help] [answered]
### ImageMagick version

7.1.1-3

### Operating system, version and so on

Ubuntu 18.04

### Description

How does one apply a colormap with mogrify?  I tried:
`mogrify -format jpg *.jp2 colormap.png -hald-clut`
And while it converted all the jp2 files to jpg, it did not apply the colormap.

**Comment by fmw42** [comment, marked as answer]
You cannot use mogrify for that. You will have to write a script FOR loop over your images and do that processing.
